### ISSUE TITLE
Can't have the same options specified for multiple subcommands

### ISSUE THREAD

**Issue by Xe** [fenekku/commandeer#4]
``` nimrod
commandline:
  subcommand codecmd, "code":
    arguments name, string
    exitoption "help", "h", codeUsage

  subcommand newSecretcmd, "newSecret":
    arguments newSecretName, string
    exitoption "help", "h", newSecretUsage

  subcommand lscmd, "ls":
    exitoption "help", "h", lsUsage

  exitoption "help", "h", usageHelp
```

``` console
$ totptool --help                                 
Usage: totptool code <name>

This will generate a TOTP code for the given secret
name in the secret store.
```

Code is [here](https://git.xeserv.us/xena/totptool). Is this intended?

**Comment by fenekku** [comment]
It is known! I have been thinking about a solution to this for a while now and I need to start implementing it. Indeed I do not want it to be that way. Thanks for the note.

**Comment by fenekku** [comment]
Fixed for version 0.10.2

Thanks again!
